driver_scripts/driver_grid.hera.sh

```diff
@@ -83,14 +83,16 @@ elif [ $gtype = stretch ]; then
   export target_lat=35.5       # Center latitude of the highest resolution tile
 elif [ $gtype = nest ] || [ $gtype = regional_gfdl ]; then
```

Comment:
I would set the default for add_lake to be false.  And I would include a lake_cutoff variable.  Please update the script prolog for these changes.   The other four 'driver' scripts will need to be updated as well.